Resources/Prototypes/Entities/Structures/Specific/Anomaly/anomaly_injectors.yml

```diff
@@ -27,6 +27,9 @@
     whitelist:
       tags:
       - AnomalyHost
+  - type: Psionic #Nyano - Summary: makes psionic on creation.
+  - type: GlimmerSource #Nyano - Summary: makes this a potential source of Glimmer.
+    active: false
```

Comment:
pro this won't do anything, this is just the injector
putting it in `AnomalyInjectionBase` should just work

Reply:
skull emoji. fixing it in a moment